help/en/html/Sidebar

```diff
@@ -42,6 +42,7 @@
             <li><dfn><a href="../package/jmri/jmrit/dispatcher/Dispatcher.shtml">Dispatcher</a></dfn>
 		    <li><dfn><a href="tools/Routes.shtml">Routes</a></dfn>
 		    <li><dfn><a href="tools/fastclock/index.shtml">Fast Clocks</a></dfn>
+		    <li><dfn><a href="tools/uss/index.shtml">US CTC</a></dfn>
```

Comment:
This should probably be USS CTC; the USS stands for Union Switch and Signal, who makes a particular style of CTC panel. (The other well-known American one is GRS, General Relay and Signal)